controllers/recipeController.js

```diff
@@ -57,6 +62,21 @@ const getSingleRecipe = async (req, res) => {
 */
 const createRecipe = async (req, res) => {
 
+    console.log("inside create recipe")
```

Comment:
console log always good for local dev testing, as it will log in the terminal window you used to npm start the application. Note, this will not help you debug in a dev/qa/prod env unless you have access to those logs on the VMs they are running on.